src/snowflake/snowpark/modin/plugin/_internal/apply_utils.py

```diff
@@ -614,15 +612,19 @@ def end_partition(self, df: native_pd.DataFrame):  # type: ignore[no-untyped-def
         # ...then the data columns for the input dataframe or series.
         *input_data_column_types,
     ]
+    # Random number generator to create a number to append to column names for UDTF.
```

Comment:
In OrderedDataFrame, we can use `generate_snowflake_quoted_identifiers` to generate a unique id to use to avoid duplicates.

Reply:
I thought about using that but I don't have access to any OrderedDataFrame in `apply_utils.py`. I can try to add a check for duplicates here to prevent further collision.

Reply:
nvm I can use `generate_snowflake_quoted_identifiers_helper`